sources/platform/console/settings.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Settings
+description: Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.
+sidebar_position: 3
+category: platform
+slug: /console/settings
+---
+
+**Learn about the Apify Console settings for the account, integrations, organization management, and notification preferences.**
+
+---
+
+## Account
+
+By clicking the **Settings** tab on the side menu, by default, you will be presented with an Account page where you can view & edit various settings regarding your account, such as:
+
+* account email
+* username
+* profile information
+* theme
+* login information
+* account delete
```

Comment:
Suggestion: Maybe instead of showing this as a list, I'd turn it into a sentence.

Reply:
I think that list is nice more visual way of representing this information, it just